### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by lilsweetcaligula**
Fixes constraint violations when tested against RDBMS-based plugins.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,25 @@ npm install seneca-store-test
 The [Senecajs org][] encourage open participation. If you feel you can help in any way, be it with
 documentation, examples, extra testing, or new features please get in touch.
 
+## Before you test
```
Comment (lilsweetcaligula) [inline README.md:50]: As requested by: https://github.com/senecajs/seneca-mongo-store/pull/84#discussion_r633458458

File: store-test.js [hunk 2]
```diff
@@ -1920,6 +1831,26 @@ function upserttest(settings) {
 
         describe('edge cases', () => {
           describe('bombarding the store with near-parallel upserts', () => {
+            //
+            // NOTE: WARNING: Chances are, in order to pass this test, you
```
Comment (lilsweetcaligula) [inline store-test.js:1835]: As requested by: https://github.com/senecajs/seneca-mongo-store/pull/84#discussion_r633458458
